docs/user_guide/SIMP_Administration/Classification_and_Data/SIMP_Compliance_Engine.inc

```diff
@@ -0,0 +1,130 @@
+.. _SIMP Compliance Engine:
+
+The SIMP Compliance Engine
+--------------------------
+
+The :term:`SIMP Compliance Engine` allows users to enable both Puppet
```

Comment:
Remove the word `both` in this line; the `as well as` on the next line makes it redundant.  Also, it's awkward/confusing to read a sentence with two separate `both` clauses.